graph/src/cypher.rs

```diff
@@ -797,6 +797,27 @@ impl<'a> Parser<'a> {
         }
     }
 
+    /// Recursively checks if a node or its descendants contain an aggregate function
+    fn contains_nested_aggregate(
```

Comment:
don't use recursion use the orx-tree  crate walk DFS/BFS option

Reply:
done